src/components/back-to-top/_macro.njk

```diff
@@ -0,0 +1,15 @@
+{% from "components/icon/_macro.njk" import onsIcon %}
+
+{% macro onsBackToTop(params) %}
+    <div class="ons-back-to-top ons-js-btt ons-back-to-top__enabled">
```

Comment:
`ons-js-btt` should be `ons-js-back-to-top` to be consistent with the other class names